specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/StaticSites.json

```diff
@@ -3811,6 +4562,164 @@
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DatabaseConnectionResponseCollection": {
+      "description": "Collection of static site database connections.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DatabaseConnectionResponse"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DatabaseConnectionPatchRequest": {
+      "description": "Static Site Database Connection Request Properties ARM resource when patching",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "DatabaseConnectionPatchRequest resource specific properties",
+          "type": "object",
+          "properties": {
+            "resourceId": {
+              "description": "The resource id of the database.",
+              "type": "string"
+            },
+            "connectionIdentity": {
+              "description": "If present, the identity is used in conjunction with ConnectionString to connect to the database. Use of the system-assigned managed identity is indicated with the string 'SystemAssigned', while use of a user-assigned managed identity is indicated with the resource id of the managed identity resource.",
+              "type": "string"
+            },
+            "connectionString": {
+              "description": "The connection string to use to connect to the database.",
+              "type": "string"
+            },
+            "region": {
+              "description": "The region of the database resource.",
+              "type": "string"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DatabaseConnectionRequest": {
+      "description": "Static Site Database Connection Request Properties ARM resource",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "DatabaseConnectionRequest resource specific properties",
+          "type": "object",
+          "properties": {
+            "resourceId": {
+              "description": "The resource id of the database.",
+              "type": "string"
+            },
+            "connectionIdentity": {
+              "description": "If provided, the identity is used in conjunction with ConnectionString to connect to the database.",
+              "type": "string"
+            },
+            "connectionString": {
+              "description": "The connection string to use to connect to the database.",
+              "type": "string"
+            },
```

Comment:
Can you please update the x-ms-secret and the readonly properties corectly everywhere for this PR : 

[https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-mutability](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-mutability)

Reply:
+1 that. Its a breaking change with old api versions... for sake of security improvement, so it should be approved for a 1 year deprecation timeline